boot/bootdata/hivesft.inf

```diff
@@ -3,26 +3,121 @@ Signature="$ReactOS$"
 
 [AddReg]
 
-HKLM,"SOFTWARE\Classes\Applications\iexplore.exe","",0x00000000,""
-HKLM,"SOFTWARE\Classes\Applications\iexplore.exe\shell\open\command","",0x00000000,"""%programfiles%\Internet Explorer\iexplore.exe"" %1"
+; Applications
+HKLM,"SOFTWARE\Classes\Applications\notepad.exe\shell\open\command","",0x00020000,"%SystemRoot%\system32\notepad.exe %1"
+HKLM,"SOFTWARE\Classes\Applications\mspaint.exe\shell\open\command","",0x00020000,"%SystemRoot%\system32\mspaint.exe %1"
+HKLM,"SOFTWARE\Classes\Applications\wordpad.exe\shell\open\command","",0x00020000,"%SystemRoot%\system32\wordpad.exe %1"
+HKLM,"SOFTWARE\Classes\Applications\iexplore.exe\shell\open\command","",0x00020000,"""%programfiles%\Internet Explorer\iexplore.exe"" %1"
```

Comment:
Ditto (since you've added `""%1""` for other iexplore.exe lines below).

Reply:
OK.